lib/submission_runners/python.rb

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+require 'submission_runners/base'
+
+module SubmissionRunners
+  class Python < Base
+    def self.image
+      'python:3.6.1'
+    end
+
+    def initialize(*args)
+      @python_builder = Docker::PythonBuilder.new(self)
+      super
+    end
+
+    delegate :build, :run, to: :@python_builder
+
+    def container_user
+      "root"
```

Comment:
Do not do this. We purposefully chose the "nobody" user because it has very limited permissions. Running unknown code in production under the root user is a massive security risk.